utils/airtable_es2015.js

```diff
@@ -161,7 +161,23 @@ var hydrateFromAirtable = (exports.hydrateFromAirtable = async function hydrateF
 var writeFeedback = (exports.writeFeedback = async function writeFeedback(
   payload
 ) {
-  var url = "https://api.airtable.com/v0/appoFDwVvNMRSaO6o/feedback";
```

Comment:
found a bug! the base key should not have been hardcoded

Reply:
ok isn't this a security a security issue now, if the key is saved on git?

Reply:
It's not a really a key in the sense that knowing it gives you access to anything. It's just a unique id for the airtable base we're using. The read/write keys are what allow you to access the base.